Shared/mods/deathmatch/logic/Utils.cpp

```diff
@@ -92,6 +92,24 @@ bool IsValidFilePath(const char* szDir)
     return true;
 }
 
+bool IsValidFilePath(const char* szDir, size_t length)
+{
+    if (szDir == nullptr)
+        return false;
+
+    unsigned char c, c_d;
```

Comment:
Please use `std::uint8_t` instead of `unsigned char`

Reply:
A pretty strange choice when we are dealing with characters from `const char*` (unsigned char is fine).